javascript/slides/varsfunctions.html

```diff
@@ -1,513 +1,528 @@
 <pre>
-function addNumbers(num1, num2) {
-  var result = num1 + num2;
-  return result; // Anything after this line won't be executed
-}
+    function addNumbers(num1, num2) {
+        var result = num1 + num2;
+        return result; // Anything after this line won't be executed
+    }
 
-var sum = addNumbers(5, 2);
+    var sum = addNumbers(5, 2);
 </pre>
 <p>You can use function calls in expressions:</p>
 <pre>
-var biggerSum = addNumbers(2, 5) + addNumbers(3, 2);
+    var biggerSum = addNumbers(2, 5) + addNumbers(3, 2);
 </pre>
 
 <p>You can even call functions inside function calls:</p>
 <pre>
-var hugeSum = addNumbers(addNumbers(5, 2), addNumbers(3, 7));
+    var hugeSum = addNumbers(addNumbers(5, 2), addNumbers(3, 7));
 </pre>
 
-<p class="speaker-notes" style="display:none;">
-Functions can be used to do processing tasks, and they can also return a value to the function caller. To do this we use the 'return' keyword. 'Return' also quits the function so it must always be the last statement in your function.
-If we assign a function call to a variable, it's value will be the result of running the function, ie whatever is 'returned'. If there is no return statement the variable's value will be 'undefined'.
-</p>
-     </section>
-    </div>
-    
-   
-    
-    
-    
+                                    <p class="speaker-notes" style="display:none;">
+                                        Functions can be used to do processing tasks, and they can also return a value to the function caller. To do this we use the 'return' keyword. 'Return' also quits the function so it must always be the last statement in your function.
+                                        If we assign a function call to a variable, it's value will be the result of running the function, ie whatever is 'returned'. If there is no return statement the variable's value will be 'undefined'.
+                                    </p>
+                                </section>
+                            </div>
+
 
-    
-   
-    
-    
-    
-    <div class="slide normal"> 
-     <header><h1>Variable Scope</h1></header> 
-     <section class="content"> 
-     <p>JS Variables have "function scope". They are visible in the function where they're defined:</p>
 
-<p>A variable with "local" scope:</p>
+
+
+
+
+
+
+
+
+                            <div class="slide normal"> 
+                               <header><h1>Variable Scope</h1></header> 
+                               <section class="content"> 
+                                   <p>JS Variables have "function scope". They are visible in the function where they're defined:</p>
+
+                                   <p>A variable with "local" scope:</p>
 <pre>
-function addNumbers(num1, num2) {
-  var localResult = num1 + num2;
-  console.log("The local result is: " + localResult);
-}
-addNumbers(5, 7);
-console.log(localResult);
+    function addNumbers(num1, num2) {
```

Comment:
extra tab snuck in here 
<img width="934" alt="screen shot 2016-05-14 at 8 20 56 am" src="https://cloud.githubusercontent.com/assets/2753545/15269198/d43ff462-19ac-11e6-8093-a623251cfa1c.png">